tests/src/rules/order.js

```diff
@@ -482,7 +523,7 @@ ruleTester.run('order', rule, {
         message: '`async` import should occur before import of `./sibling`',
       }, {
         ruleId: 'order',
-        message: '`fs` import should occur before import of `./sibling`',
+        message: '`fs` import should occur before import of `async`',
```

Comment:
So the original order is:
```
sibling
async
fs
```
And I believe the desired order is
```
fs
async
sibling
```
So saying that `fs` should be before `./sibling` is technically incorrect, as it leaves out the additional requirement that `fs` also appear before `async`. 

Do you agree?